privilege/privileges/privileges.go

```diff
@@ -505,15 +596,19 @@ func (p *UserPrivileges) ConnectionVerification(user *auth.UserIdentity, authUse
 	if err != nil {
 		return err
 	}
-
-	p.user = authUser
-	p.host = record.Host
 	if sandBoxMode {
 		return &ErrInSandBoxMode{}
 	}
 	return nil
 }
 
+// AuthSuccess implements the Manager interface.
+// Auth Success state
```

Comment:
Please use a complete sentence to make the comment clear.

Reply:
done